bin/bintray/upload

```diff
@@ -138,6 +148,7 @@ do
 		r) comp_release="$OPTARG" ;;
 		p) comp_prerelease="$OPTARG" ;;
 		a) arch="$OPTARG" ;;
+		n) dry_run=true ;;
```

Comment:
.... how many times did you re-read this line just in case? :-P

Reply:
Hopefully only once now that we have tests :wink: